packages/ra-ui-materialui/src/list/Pagination.spec.js

```diff
@@ -43,19 +43,50 @@ describe('<Pagination />', () => {
             expect(queryByText('ra.navigation.no_results')).toBeNull();
         });
 
-        it('should not display a pagination limit on an out of bounds page', () => {
+        it('should display a pagination limit on an out of bounds page (more than total pages)', async () => {
```

Comment:
why changing this title ?
the previous was true, we not display the pagination limit component, we display the last page